t/t1018-mf-priority-disable-entry.t

```diff
@@ -44,7 +44,7 @@ test_expect_success 'submit a job successfully under default bank' '
 	jobid1=$(flux mini submit -n1 hostname) &&
 	flux job wait-event -f json $jobid1 priority | jq '.context.priority' > job1.test &&
 	cat <<-EOF >job1.expected &&
-	10050000
+	50000
```

Comment:
It might be useful to state in the commit message why the priority changes are a result of the changes in validating queues since, at least to me, that is not an obvious effect. Though please correct me if this should be clear.

Reply:
Yeah, looking back I realize this isn't as obvious as I initially thought, sorry about that.

The reason the priority changes for these jobs is because the queue factor is `UNSPECIFIED_QUEUE`, which is 0. This changes the resulting integer priority of the job from `1005000` to just `50000`, as before, the multi-factor priority plugin had a `"default"` queue entry (and a factor of `1000`) when an association submitted jobs under a default queue (like in the test cases above). I should make note of this in the commit message, and will push up a fix.